Engine/Component/ComponentBillboard.cpp

```diff
@@ -12,25 +16,23 @@
 ComponentBillboard::ComponentBillboard() : Component(nullptr, ComponentType::BILLBOARD)
 {
 	InitData();
+	ChangeBillboardType(AlignmentType::VIEW_POINT);
```

Comment:
put the default aligment type to viewpoint instead by default on the h

Reply:
It's done, but his method does more stuff like